libraries-bom/pom.xml

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>libraries-bom</artifactId>
+  <version>25.3.1-SNAPSHOT</version><!-- {x-version-update:libraries-bom:current} -->
+  <packaging>pom</packaging>
+
+  <name>Google Cloud Platform Supported Libraries</name>
+  <description>
+    A compatible set of Google Cloud open source libraries.
+    Document: https://cloud.google.com/java/docs/bom
+  </description>
+  <url>https://cloud.google.com/java/docs/bom</url>
+  <organization>
+    <name>Google LLC</name>
+    <url>https://cloud.google.com</url>
+  </organization>
+  <inceptionYear>2019</inceptionYear>
+  <developers>
+    <developer>
+      <name>Elliotte Rusty Harold</name>
+    </developer>
+  </developers>
+
+  <issueManagement>
+    <url>https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues</url>
+  </issueManagement>
+  <scm>
+    <connection>scm:git:git@github.com:GoogleCloudPlatform/cloud-opensource-java.git</connection>
+    <developerConnection>scm:git:git@github.com:GoogleCloudPlatform/cloud-opensource-java.git
+    </developerConnection>
+    <url>https://github.com/GoogleCloudPlatform/cloud-opensource-java/boms/cloud-oss-bom</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <guava.version>31.1-jre</guava.version>
+    <gson.version>2.9.0</gson.version>
+    <google.cloud.core.version>2.6.0</google.cloud.core.version>
+    <io.grpc.version>1.45.1</io.grpc.version>
+    <http.version>1.41.7</http.version>
+    <protobuf.version>3.19.4</protobuf.version>
+    <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
+        When updating gax.version, update gax.httpjson.version too. -->
+    <gax.version>2.16.0</gax.version>
+    <gax.httpjson.version>0.101.0</gax.httpjson.version>
+    <auth.version>1.6.0</auth.version>
+    <api-common.version>2.1.5</api-common.version>
+    <common.protos.version>2.8.3</common.protos.version>
+    <iam.protos.version>1.3.1</iam.protos.version>
```

Comment:
I envision these properties will be gone once we start using the first-party-dependencies BOM (a part of the shared dependencies BOM). But for now I focus on the move of the pom to this repository.

The first-party-dependencies change will happen after the successful release of the Libraries BOM from this repository.